app/indexers/embargo_metadata_indexer.rb

```diff
@@ -1,27 +1,27 @@
 # frozen_string_literal: true
 
 class EmbargoMetadataIndexer
-  attr_reader :resource
+  attr_reader :cocina
 
-  def initialize(resource:, **)
-    @resource = resource
+  def initialize(cocina:, **)
+    @cocina = cocina
   end
 
   # These fields are used by the EmbargoReleaseService in dor-services-app
   # @return [Hash] the partial solr document for embargoMetadata
   def to_solr
-    {
-      'embargo_status_ssim' => embargo_status
-    }.tap do |solr_doc|
-      solr_doc['embargo_release_dtsim'] = Array(release_date.first.utc.strftime('%FT%TZ')) if release_date.first.present?
+    {}.tap do |solr_doc|
+      embargo_release_date = embargo_release_date(cocina)
+      if embargo_release_date.present?
+        solr_doc['embargo_status_ssim'] = ['embargoed']
+        solr_doc['embargo_release_dtsim'] = [embargo_release_date.utc.strftime('%FT%TZ')]
```

Comment:
💨 I think you can do `embargo_release_date.utc.iso8601` here and it is equivalent and maybe easier to understand. 🤷

Reply:
changed - thanks.  I thought I ran into trouble with the milliseconds included in iso8601 but apparently I misremembered.